sass/_SmartSnippet.scss

```diff
@@ -1,40 +1,158 @@
 @import 'Variables';
 
+@mixin triangle($size: 20px) {
+  position: relative;
+  &::before {
+    content: '';
+    display: inline-block;
+    width: $size;
+    height: $size;
+    background-color: $color-blue-background;
+    position: absolute;
+    left: 50%;
+    transform: translate(-50%, 110%) rotate(45deg);
+  }
+}
+
+@mixin yes-no-button($type) {
```

Comment:
The mixin makes it nice to read and use for us, but it generates very deeply nested rules, making them much more annoying to customize by customers.

It's generally better for the clients if we can achieve super flat rules (with longer names if need be).

This comment applies to a lot of styling in here. Most of them could be put flat/top level without embedding in one another, unless it's about "meta classes", like `:hover` and such.

Reply:
Good point, I'll try to flatten it further.

Reply:
Update: I didn't know, but SASS has a `@at-root` symbol. I'll focus on the readability of the CSS and then add `@at-root` symbols to flatten the output as much as possible.